CMakeLists.txt

```diff
@@ -22,7 +22,12 @@ endif()
 # CUDA
 ###############################################################################
 find_package(CUDA REQUIRED)
-set(CUDA_NVCC_FLAGS "-arch=sm_20;-use_fast_math;")
+if(CUDA_VERSION VERSION_LESS 9.0)
```

Comment:
Please add a `if(NOT DEFINED COMPUTE_CAPABILITY)` around this, so one can configure it externally.